Doppler.Sap/appsettings.json

```
@@ -35,7 +36,9 @@
           "Endpoint": "BusinessPartners"
         },
         "BillingConfig": {
-          "Endpoint": "Invoices"
+          "Endpoint": "Invoices",
+          "NeedCreateIncomingPayments": "true",
```

You are using a string for a boolean value.

```suggestion
          "NeedCreateIncomingPayments": true,
```

---

yes, my bad. I will update it.

---

Done